github_connector/models/abstract_github_model.py

```diff
@@ -141,6 +141,10 @@ def get_from_id_or_create(self, data, extra_data=None):
                         data[self._github_login_field], data['id'],
                         self.github_type()))
                 return existing_object
+           elif len(existing_object) > 1:
+                _logger.error(
+                    "Duplicate object with github login %s",
+                    data[self._github_login_field])
```

Comment:
Hi @sbidoul,
thanks for this improvment, usefull when there are some duplicates in the DB.
I think you should raise an error. Otherwise, l152 will try to create a new obj, don't you think ?

Reply:
Indeed. I changed it to a UserError.